### PR TITLE
change buttons links

### PR DESCRIPTION
**Pull request by developerfred**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -315,5 +309,13 @@ <h1>Make a Pledge:</h1>
     document.location.href="https://twitter.com/intent/tweet?text=" +encodeURI(text)+ "&url=" + encodeURI(url);
   }
   
+
+  document.getElementById('hardcover').onclick = function(e){
+    document.location.href="https://www.amazon.com.br/Infinite-Machine-Crypto-hackers-Building-Internet/dp/0062886142/ref=tmm_hrd_swatch_0?_encoding=UTF8&qid=1644880269&sr=8-7"
```
Comment (owocki) [inline index.html]: these are the wrong lniks.  pls defer to whatever is in the a href

File: index.html [hunk 2]
```diff
@@ -315,5 +309,13 @@ <h1>Make a Pledge:</h1>
...
+  }
+  
+  document.getElementById('epub').onclick = function(e){
+    document.location.href="https://www.amazon.com.br/Infinite-Machine-Crypto-hackers-Building-Internet-ebook/dp/B07X8HS2WC/ref=tmm_kin_swatch_0?_encoding=UTF8&qid=1644880269&sr=8-7"
```
Comment (owocki) [inline index.html]: these are the wrong lniks.  pls defer to whatever is in the a href

File: index.html (hunk 1 above)
Comment (developerfred) [reply index.html]: fixed to  document.location.href="#"

File: index.html (hunk 2 above)
Comment (developerfred) [reply index.html]: fixed to  document.location.href="#"

**Comment by owocki** [comment]
7af4ff5a3844aff976604610dd1586df2cc4be8b

**Pull request closed without merging**